docs/pages/administration/registry.rst

```diff
@@ -0,0 +1,37 @@
+.. _nuts-registry-administration:
```

Comment:
isn't this page part of the registry? Tag + filename?

Reply:
:facepalm: